diff --git a/frontend/pages/reports/label-generator.vue b/frontend/pages/reports/label-generator.vue
index 81e92c4d..aef88775 100644
--- a/frontend/pages/reports/label-generator.vue
+++ b/frontend/pages/reports/label-generator.vue
@@ -439,7 +439,7 @@
           </div>
           <div class="ml-2 flex flex-col justify-center">
             <div class="font-bold">{{ item.assetID }}</div>
-            <div class="text-xs font-light italic">HomeBox</div>
+            <div class="text-xs font-light italic">Fachschaft IM</div>
             <div class="overflow-hidden text-wrap text-xs">{{ item.name }}</div>
             <div class="text-xs">{{ item.location }}</div>
           </div>
